site-src/geps/gep-713.md

```diff
@@ -1,25 +1,35 @@
-# GEP-713: Policy Attachment
+# GEP-713: Metaresources and Policy Attachment
```

Comment:
I'm a little torn on this, but I'm wondering if we should have a new GEP to describe this new form of policy attachment. That would allow either the hierarchical or direct versions to graduate independently if necessary. In any case we should have unified docs, and if the base API(s) are nearly identical, they probably do belong in the same GEP, just figured I'd least mention the alternative.

Reply:
I've thought about this, and I think that having them both in the same document is better - I think that the uses of direct policy attachment will be much less that hierarchical policy attachment.